### DISCUSSION TITLE
I have a problem with my wallet

### DISCUSSION THREAD

**Discussion by heshmatkhan** [Chia-Network/chia-blockchain#8509] [Support]
Hi
I transferred some chia to my wallet  from coinex 11 days ago.
But the wallet balance is zero

Both the wallet and the full node are synced

https://xchscan.com/txns/0xc4af09c13b830dc0734a3bbb5fcd96e85eca6c0d554926cf0824a37eb12bd313

What is the problem?

**Comment by loppefaaret** [comment]
hmm, you got the address from the gui ? do you have multiple keys on the system ?

**Reply by heshmatkhan** [reply]
I only have one key on the system

**Comment by heshmatkhan** [comment]
<a href="https://ibb.co/6JBV4Kq"><img src="https://i.ibb.co/tPXkb01/chia-1.png" alt="chia-1" border="0"></a>
<a href="https://ibb.co/Jr3bLRJ"><img src="https://i.ibb.co/Zc2ws6D/chia-2.png" alt="chia-2" border="0"></a>

**Reply by loppefaaret** [reply]
once you get both the full node and the wallet synced, there is XCH on the address:
https://xchscan.com/address/xch1qnt9utqq4mekxny7ptf784g87sdpplkynz7pdlwtnnewq9rw6a3sttct3g

**Reply by heshmatkhan** [reply]
Thank you

**Comment by BrandtH22** [comment]
@heshmatkhan  , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
